src/pages/staticPages/datasetView/DatasetPage.tsx

```diff
@@ -21,7 +21,7 @@ export class DatasetPageStore {
     });
 
     readonly downloadList = remoteData(() => {
-        if (AppConfig.serverConfig.app_name === 'public-portal') {
+        if (AppConfig.serverConfig.show_study_download_list) {
```

Comment:
this one is maybe ok to leave as it was as well, not sure what exactly to call this prop. It's like some way to only list studies that are also on datahub (https://github.com/cBioPortal/datahub/tree/master/public). Not really something anyone else would use